### PR TITLE
libvdpau 1.4

### PR DESCRIPTION
**Pull request by morusn**
switch to `meson` and updated to 1.4

### CODE REVIEW

**Comment by carlocab** [comment]
The build system was also switched to `meson`.

**Comment by morusn** [comment]
So I tried to switch it to `meson`
I have no idea how to properly write `test`

File: Formula/libvdpau.rb [hunk 1]
```diff
@@ -17,25 +17,21 @@ class Libvdpau < Formula
     sha256 mojave:        "59980ec6bf90b676354ddda5e3c93a6240c4564d1c01aa35b1f1aa804d7b949a"
   end
 
-  depends_on "pkg-config" => [:build, :test]
+  depends_on "meson" => [:build]
+  depends_on "ninja" => [:build]
+  depends_on "pkg-config" => [:build]
```
Comment (carlocab) [inline Formula/libvdpau.rb:22]:
```suggestion
  depends_on "meson" => :build
  depends_on "ninja" => :build
  depends_on "pkg-config" => [:build, :test]
```

I think you still need `pkg-config` for the test.

File: Formula/libvdpau.rb [hunk 2]
```diff
@@ -17,25 +17,21 @@ class Libvdpau < Formula
...
-                          "--sysconfdir=#{etc}",
-                          "--disable-dependency-tracking",
-                          "--disable-silent-rules",
-                          "--enable-dri2"
-    system "make"
-    system "make", "install"
   end
-
   test do
-    assert_match "-I#{include}", shell_output("pkg-config --cflags vdpau")
```
Comment (carlocab) [inline Formula/libvdpau.rb:39]: Does this test no longer work?

Comment (morusn) [reply Formula/libvdpau.rb:39]: Oh, it does work, thanks

**Review by SMillerDev — APPROVED** [review]
Thanks @morusn  ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Pull request closed without merging**